ibm/conns/config.go

```diff
@@ -1809,10 +1802,18 @@ func (c *Config) ClientSession() (interface{}, error) {
 	session.apigatewayAPI = apigatewayAPI
 
 	// POWER SYSTEMS Service
-	ibmpisession, err := ibmpisession.New(sess.BluemixSession.Config.IAMAccessToken, c.Region, false, session.bmxUserDetails.UserAccount, c.Zone)
+	piURL := ContructEndpoint(c.Region, "power-iaas.cloud.ibm.com")
+	ibmPIOptions := &ibmpisession.IBMPIOptions{
+		Authenticator: authenticator,
+		Debug:         os.Getenv("TF_LOG") != "",
+		Region:        c.Region,
+		URL:           EnvFallBack([]string{"IBMCLOUD_PI_API_ENDPOINT"}, piURL),
```

Comment:
should it be `IBMCLOUD_POWER_API_ENDPOINT` or `IBMCLOUD_PI_API_ENDPOINT`?

Reply:
`IBMCLOUD_POWER_API_ENDPOINT` is already taken by the sdk. Hence using `IBMCLOUD_PI_API_ENDPOINT` for Terraform provider.